### PR TITLE
Feature/external rest

### PR DESCRIPTION
**Pull request by moahmo**
- Added basic methods for data retrieval, with notable absence of field filtering (once I settle on a design, it will be ddded)
- Basic change method which conforms to the RFC documents, but nothing fancy
- Tried to add as much tests as possible in this phase

### CODE REVIEW

File: src/api/connector/external/rest.js [hunk 1]
```diff
@@ -1,16 +1,101 @@
...
+    throw new Error('Identifier field not found in params');
+  }
+
+  return identifier;
+};
+
+const getSortPrefix = (sortBy) => {
+  let sortPrefix = '';
+
+  if (sortBy === 'desc') sortPrefix = '-';
```
Comment (manico) [inline src/api/connector/external/rest.js]: Do we need casing normalisation here?

Comment (moahmo) [reply src/api/connector/external/rest.js]: Casing normalization for order param, e.g. `Desc`? It won't hurt if I add it.

Comment (manico) [reply src/api/connector/external/rest.js]: Yup. Just use lodash toLower there. :)

File: src/api/connector/external/rest.js [hunk 2]
```diff
@@ -1,16 +1,102 @@
...
+  else if (sortByParam === 'asc') sortPrefix = '+';
+
+  return sortPrefix;
+};
+
+const getApiParams = (clientParams) => {
+  const apiParams = {};
+
+  // TODO: Add field filter handling
+  apiParams.sort = `${getSortPrefix(clientParams.sortBy)}${clientParams.sort}`;
```
Comment (Lanchi) [inline src/api/connector/external/rest.js]: So, client params would be:
```
{
   sort: 'age',
   sortBy: 'asc'
}
```

Wouldn't it be more straightforward to change these two prop names so `sortBy` has the value of sorting param, and `sort` has sorting direction? 🤔

Comment (moahmo) [reply src/api/connector/external/rest.js]: Yeah, more _straightforward_ could be used to describe it, but I'd say that more _correct_ is even more appropriate :)

Comment (Lanchi) [reply src/api/connector/external/rest.js]: Me at my subtlest 😄

**Pull request merged**